classes/production/laker-polling/application.yml

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
Delete the classes folder from your repo. It should be getting ignored but it looks like this repo didn't initially have the .gitignore file in it.